sources/packages/backend/libs/services/src/queue/queue.service.ts

```diff
@@ -65,6 +75,14 @@ export class QueueService {
    */
   async getQueueConfiguration(queueName: QueueNames): Promise<Bull.JobOptions> {
     const queueConfig = await this.queueConfigurationDetails(queueName);
+    // If a queue configuration record is inactive due to its false in_active flag
```

Comment:
What is the motivation for this constraint?
The method is intended to return a queue configuration and it can still return it even when inactive.